Havit.Blazor.Components.Web.Bootstrap/Forms/HxMultiSelect.cs

```diff
@@ -110,6 +110,11 @@ public class HxMultiSelect<TValue, TItem> : HxInputBase<List<TValue>>, IInputWit
 	/// </summary>
 	[Parameter] public RenderFragment InputGroupEndTemplate { get; set; }
 
+	[Parameter] public bool EnableFiltering { get; set; }
```

Comment:
We don't have any other `EnableXy` parameters for optional features.
This should probably be `Filterable="true"` or `AllowFiltering="true"` or `Filtering="true"`. Not sure now. :-D
(We can change this later before merging to master.)